src/commands/validate.ts

```diff
@@ -35,4 +61,26 @@ export default class Validate extends Command {
       process.exitCode = 1;
     }
   }
+
+  private async calculateScore(filePath:string) {
```

Comment:
I suggest to extract the score logic and move it under `models` or `utils` for now. It will make this https://github.com/asyncapi/cli/pull/1200 easy.

Reply:
Moved to Utils Thanks